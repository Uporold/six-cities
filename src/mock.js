// eslint-disable-next-line import/prefer-default-export
export const fakeAdTitles = [
  `Beautiful & luxurious apartment at great location`,
  `Wood and stone place`,
  `Canal View Prinsengracht`,
  `Nice, cozy, warm big bed apartment`,
];
