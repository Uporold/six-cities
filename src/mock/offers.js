// eslint-disable-next-line import/prefer-default-export
export const hotels = [
  {
    city: {
      name: "Brussels",
      location: { latitude: 50.846557, longitude: 4.351697, zoom: 13 },
    },
    previewImage:
      "https://htmlacademy-react-3.appspot.com/six-cities/static/hotel/15.jpg",
    images: [
      "https://htmlacademy-react-3.appspot.com/six-cities/static/hotel/2.jpg",
      "https://htmlacademy-react-3.appspot.com/six-cities/static/hotel/5.jpg",
      "https://htmlacademy-react-3.appspot.com/six-cities/static/hotel/3.jpg",
      "https://htmlacademy-react-3.appspot.com/six-cities/static/hotel/13.jpg",
      "https://htmlacademy-react-3.appspot.com/six-cities/static/hotel/16.jpg",
      "https://htmlacademy-react-3.appspot.com/six-cities/static/hotel/9.jpg",
      "https://htmlacademy-react-3.appspot.com/six-cities/static/hotel/17.jpg",
      "https://htmlacademy-react-3.appspot.com/six-cities/static/hotel/6.jpg",
      "https://htmlacademy-react-3.appspot.com/six-cities/static/hotel/15.jpg",
      "https://htmlacademy-react-3.appspot.com/six-cities/static/hotel/14.jpg",
      "https://htmlacademy-react-3.appspot.com/six-cities/static/hotel/20.jpg",
      "https://htmlacademy-react-3.appspot.com/six-cities/static/hotel/8.jpg",
      "https://htmlacademy-react-3.appspot.com/six-cities/static/hotel/7.jpg",
      "https://htmlacademy-react-3.appspot.com/six-cities/static/hotel/4.jpg",
    ],
    title: "Nice, cozy, warm big bed apartment",
    isFavorite: false,
    isPremium: false,
    rating: 4.9,
    type: "apartment",
    bedrooms: 3,
    maxAdults: 6,
    price: 488,
    goods: [
      "Breakfast",
      "Baby seat",
      "Washer",
      "Towels",
      "Fridge",
      "Air conditioning",
      "Laptop friendly workspace",
    ],
    host: {
      id: 25,
      name: "Angelina",
      isPro: true,
      avatarUrl: "img/avatar-angelina.jpg",
    },
    description:
      "Peaceful studio in the most wanted area in town. Quiet house Near of everything. Completely renovated. Lovely neighbourhood, lot of trendy shops, restaurants and bars in a walking distance.",
    location: {
      latitude: 52.3909553943508,
      longitude: 4.85309666406198,
      zoom: 16,
    },
    id: 1,
  },
  {
    city: {
      name: "Paris",
      location: { latitude: 48.85661, longitude: 2.351499, zoom: 13 },
    },
    previewImage:
      "https://htmlacademy-react-3.appspot.com/six-cities/static/hotel/12.jpg",
    images: [
      "https://htmlacademy-react-3.appspot.com/six-cities/static/hotel/5.jpg",
      "https://htmlacademy-react-3.appspot.com/six-cities/static/hotel/10.jpg",
      "https://htmlacademy-react-3.appspot.com/six-cities/static/hotel/8.jpg",
      "https://htmlacademy-react-3.appspot.com/six-cities/static/hotel/16.jpg",
      "https://htmlacademy-react-3.appspot.com/six-cities/static/hotel/4.jpg",
      "https://htmlacademy-react-3.appspot.com/six-cities/static/hotel/9.jpg",
      "https://htmlacademy-react-3.appspot.com/six-cities/static/hotel/14.jpg",
      "https://htmlacademy-react-3.appspot.com/six-cities/static/hotel/17.jpg",
      "https://htmlacademy-react-3.appspot.com/six-cities/static/hotel/2.jpg",
      "https://htmlacademy-react-3.appspot.com/six-cities/static/hotel/15.jpg",
      "https://htmlacademy-react-3.appspot.com/six-cities/static/hotel/20.jpg",
      "https://htmlacademy-react-3.appspot.com/six-cities/static/hotel/13.jpg",
      "https://htmlacademy-react-3.appspot.com/six-cities/static/hotel/12.jpg",
      "https://htmlacademy-react-3.appspot.com/six-cities/static/hotel/19.jpg",
    ],
    title: "Penthouse, 4-5 rooms + 5 balconies",
    isFavorite: false,
    isPremium: false,
    rating: 2.9,
    type: "house",
    bedrooms: 3,
    maxAdults: 9,
    price: 327,
    goods: [
      "Baby seat",
      "Washer",
      "Towels",
      "Laptop friendly workspace",
      "Dishwasher",
      "Fridge",
      "Air conditioning",
      "Coffee machine",
      "Washing machine",
      "Breakfast",
    ],
    host: {
      id: 25,
      name: "Angelina",
      isPro: true,
      avatarUrl: "img/avatar-angelina.jpg",
    },
    description:
      "I am happy to welcome you to my apartment in the city center! Three words: location, cosy and chic!",
    location: {
      latitude: 52.369553943508,
      longitude: 4.85309666406198,
      zoom: 16,
    },
    id: 2,
  },
  {
    city: {
      name: "Cologne",
      location: { latitude: 50.938361, longitude: 6.959974, zoom: 13 },
    },
    previewImage:
      "https://htmlacademy-react-3.appspot.com/six-cities/static/hotel/3.jpg",
    images: [
      "https://htmlacademy-react-3.appspot.com/six-cities/static/hotel/6.jpg",
      "https://htmlacademy-react-3.appspot.com/six-cities/static/hotel/2.jpg",
      "https://htmlacademy-react-3.appspot.com/six-cities/static/hotel/7.jpg",
      "https://htmlacademy-react-3.appspot.com/six-cities/static/hotel/9.jpg",
      "https://htmlacademy-react-3.appspot.com/six-cities/static/hotel/11.jpg",
      "https://htmlacademy-react-3.appspot.com/six-cities/static/hotel/14.jpg",
      "https://htmlacademy-react-3.appspot.com/six-cities/static/hotel/12.jpg",
      "https://htmlacademy-react-3.appspot.com/six-cities/static/hotel/3.jpg",
      "https://htmlacademy-react-3.appspot.com/six-cities/static/hotel/1.jpg",
      "https://htmlacademy-react-3.appspot.com/six-cities/static/hotel/13.jpg",
      "https://htmlacademy-react-3.appspot.com/six-cities/static/hotel/8.jpg",
      "https://htmlacademy-react-3.appspot.com/six-cities/static/hotel/16.jpg",
      "https://htmlacademy-react-3.appspot.com/six-cities/static/hotel/10.jpg",
      "https://htmlacademy-react-3.appspot.com/six-cities/static/hotel/4.jpg",
    ],
    title: "House in countryside",
    isFavorite: false,
    isPremium: false,
    rating: 3.6,
    type: "house",
    bedrooms: 5,
    maxAdults: 5,
    price: 461,
    goods: [
      "Air conditioning",
      "Towels",
      "Laptop friendly workspace",
      "Washer",
      "Fridge",
      "Baby seat",
      "Breakfast",
    ],
    host: {
      id: 25,
      name: "Angelina",
      isPro: true,
      avatarUrl: "img/avatar-angelina.jpg",
    },
    description:
      "Peaceful studio in the most wanted area in town. Quiet house Near of everything. Completely renovated. Lovely neighbourhood, lot of trendy shops, restaurants and bars in a walking distance.",
    location: {
      latitude: 52.3909553943508,
      longitude: 4.929309666406198,
      zoom: 16,
    },
    id: 3,
  },
  {
    city: {
      name: "Paris",
      location: { latitude: 48.85661, longitude: 2.351499, zoom: 13 },
    },
    previewImage:
      "https://htmlacademy-react-3.appspot.com/six-cities/static/hotel/12.jpg",
    images: [
      "https://htmlacademy-react-3.appspot.com/six-cities/static/hotel/12.jpg",
      "https://htmlacademy-react-3.appspot.com/six-cities/static/hotel/5.jpg",
      "https://htmlacademy-react-3.appspot.com/six-cities/static/hotel/7.jpg",
      "https://htmlacademy-react-3.appspot.com/six-cities/static/hotel/6.jpg",
      "https://htmlacademy-react-3.appspot.com/six-cities/static/hotel/14.jpg",
      "https://htmlacademy-react-3.appspot.com/six-cities/static/hotel/4.jpg",
      "https://htmlacademy-react-3.appspot.com/six-cities/static/hotel/13.jpg",
      "https://htmlacademy-react-3.appspot.com/six-cities/static/hotel/1.jpg",
      "https://htmlacademy-react-3.appspot.com/six-cities/static/hotel/2.jpg",
      "https://htmlacademy-react-3.appspot.com/six-cities/static/hotel/10.jpg",
      "https://htmlacademy-react-3.appspot.com/six-cities/static/hotel/3.jpg",
      "https://htmlacademy-react-3.appspot.com/six-cities/static/hotel/15.jpg",
      "https://htmlacademy-react-3.appspot.com/six-cities/static/hotel/19.jpg",
      "https://htmlacademy-react-3.appspot.com/six-cities/static/hotel/11.jpg",
    ],
    title: "Perfectly located Castro",
    isFavorite: false,
    isPremium: false,
    rating: 4.1,
    type: "apartment",
    bedrooms: 5,
    maxAdults: 9,
    price: 243,
    goods: [
      "Coffee machine",
      "Air conditioning",
      "Breakfast",
      "Baby seat",
      "Dishwasher",
      "Laptop friendly workspace",
      "Fridge",
      "Washer",
      "Towels",
    ],
    host: {
      id: 25,
      name: "Angelina",
      isPro: true,
      avatarUrl: "img/avatar-angelina.jpg",
    },
    description:
      "This is a place for dreamers to reset, reflect, and create. Designed with a 'slow' pace in mind, our hope is that you enjoy every part of your stay; from making local coffee by drip in the morning, choosing the perfect record to put on as the sun sets.",
    location: {
      latitude: 52.3809553943508,
      longitude: 4.939309666406198,
      zoom: 16,
    },
    id: 4,
  },
  {
    city: {
      name: "Brussels",
      location: { latitude: 50.846557, longitude: 4.351697, zoom: 13 },
    },
    previewImage:
      "https://htmlacademy-react-3.appspot.com/six-cities/static/hotel/18.jpg",
    images: [
      "https://htmlacademy-react-3.appspot.com/six-cities/static/hotel/5.jpg",
      "https://htmlacademy-react-3.appspot.com/six-cities/static/hotel/2.jpg",
      "https://htmlacademy-react-3.appspot.com/six-cities/static/hotel/13.jpg",
      "https://htmlacademy-react-3.appspot.com/six-cities/static/hotel/19.jpg",
      "https://htmlacademy-react-3.appspot.com/six-cities/static/hotel/9.jpg",
      "https://htmlacademy-react-3.appspot.com/six-cities/static/hotel/20.jpg",
      "https://htmlacademy-react-3.appspot.com/six-cities/static/hotel/11.jpg",
      "https://htmlacademy-react-3.appspot.com/six-cities/static/hotel/1.jpg",
      "https://htmlacademy-react-3.appspot.com/six-cities/static/hotel/18.jpg",
      "https://htmlacademy-react-3.appspot.com/six-cities/static/hotel/12.jpg",
      "https://htmlacademy-react-3.appspot.com/six-cities/static/hotel/8.jpg",
      "https://htmlacademy-react-3.appspot.com/six-cities/static/hotel/10.jpg",
      "https://htmlacademy-react-3.appspot.com/six-cities/static/hotel/14.jpg",
      "https://htmlacademy-react-3.appspot.com/six-cities/static/hotel/16.jpg",
    ],
    title: "House in countryside",
    isFavorite: false,
    isPremium: false,
    rating: 3,
    type: "house",
    bedrooms: 2,
    maxAdults: 6,
    price: 365,
    goods: ["Laptop friendly workspace"],
    host: {
      id: 25,
      name: "Angelina",
      isPro: true,
      avatarUrl: "img/avatar-angelina.jpg",
    },
    description:
      "This is a place for dreamers to reset, reflect, and create. Designed with a 'slow' pace in mind, our hope is that you enjoy every part of your stay; from making local coffee by drip in the morning, choosing the perfect record to put on as the sun sets.",
    location: { latitude: 50.842557, longitude: 4.3536969999999995, zoom: 16 },
    id: 5,
  },
];

/*
// eslint-disable-next-line import/prefer-default-export
export const hotels = [
  {
    city: {
      name: "Brussels",
      location: { latitude: 50.846557, longitude: 4.351697, zoom: 13 },
    },
    previewImage:
      "https://htmlacademy-react-3.appspot.com/six-cities/static/hotel/15.jpg",
    images: [
      "https://htmlacademy-react-3.appspot.com/six-cities/static/hotel/2.jpg",
      "https://htmlacademy-react-3.appspot.com/six-cities/static/hotel/5.jpg",
      "https://htmlacademy-react-3.appspot.com/six-cities/static/hotel/3.jpg",
      "https://htmlacademy-react-3.appspot.com/six-cities/static/hotel/13.jpg",
      "https://htmlacademy-react-3.appspot.com/six-cities/static/hotel/16.jpg",
      "https://htmlacademy-react-3.appspot.com/six-cities/static/hotel/9.jpg",
      "https://htmlacademy-react-3.appspot.com/six-cities/static/hotel/17.jpg",
      "https://htmlacademy-react-3.appspot.com/six-cities/static/hotel/6.jpg",
      "https://htmlacademy-react-3.appspot.com/six-cities/static/hotel/15.jpg",
      "https://htmlacademy-react-3.appspot.com/six-cities/static/hotel/14.jpg",
      "https://htmlacademy-react-3.appspot.com/six-cities/static/hotel/20.jpg",
      "https://htmlacademy-react-3.appspot.com/six-cities/static/hotel/8.jpg",
      "https://htmlacademy-react-3.appspot.com/six-cities/static/hotel/7.jpg",
      "https://htmlacademy-react-3.appspot.com/six-cities/static/hotel/4.jpg",
    ],
    title: "Nice, cozy, warm big bed apartment",
    isFavorite: false,
    isPremium: false,
    rating: 4.9,
    type: "apartment",
    bedrooms: 3,
    maxAdults: 6,
    price: 488,
    goods: [
      "Breakfast",
      "Baby seat",
      "Washer",
      "Towels",
      "Fridge",
      "Air conditioning",
      "Laptop friendly workspace",
    ],
    host: {
      id: 25,
      name: "Angelina",
      isPro: true,
      avatarUrl: "img/avatar-angelina.jpg",
    },
    description:
      "Peaceful studio in the most wanted area in town. Quiet house Near of everything. Completely renovated. Lovely neighbourhood, lot of trendy shops, restaurants and bars in a walking distance.",
    location: { latitude: 50.869557, longitude: 4.332697, zoom: 16 },
    id: 1,
  },
  {
    city: {
      name: "Paris",
      location: { latitude: 48.85661, longitude: 2.351499, zoom: 13 },
    },
    previewImage:
      "https://htmlacademy-react-3.appspot.com/six-cities/static/hotel/12.jpg",
    images: [
      "https://htmlacademy-react-3.appspot.com/six-cities/static/hotel/5.jpg",
      "https://htmlacademy-react-3.appspot.com/six-cities/static/hotel/10.jpg",
      "https://htmlacademy-react-3.appspot.com/six-cities/static/hotel/8.jpg",
      "https://htmlacademy-react-3.appspot.com/six-cities/static/hotel/16.jpg",
      "https://htmlacademy-react-3.appspot.com/six-cities/static/hotel/4.jpg",
      "https://htmlacademy-react-3.appspot.com/six-cities/static/hotel/9.jpg",
      "https://htmlacademy-react-3.appspot.com/six-cities/static/hotel/14.jpg",
      "https://htmlacademy-react-3.appspot.com/six-cities/static/hotel/17.jpg",
      "https://htmlacademy-react-3.appspot.com/six-cities/static/hotel/2.jpg",
      "https://htmlacademy-react-3.appspot.com/six-cities/static/hotel/15.jpg",
      "https://htmlacademy-react-3.appspot.com/six-cities/static/hotel/20.jpg",
      "https://htmlacademy-react-3.appspot.com/six-cities/static/hotel/13.jpg",
      "https://htmlacademy-react-3.appspot.com/six-cities/static/hotel/12.jpg",
      "https://htmlacademy-react-3.appspot.com/six-cities/static/hotel/19.jpg",
    ],
    title: "Penthouse, 4-5 rooms + 5 balconies",
    isFavorite: false,
    isPremium: false,
    rating: 2.9,
    type: "house",
    bedrooms: 3,
    maxAdults: 9,
    price: 327,
    goods: [
      "Baby seat",
      "Washer",
      "Towels",
      "Laptop friendly workspace",
      "Dishwasher",
      "Fridge",
      "Air conditioning",
      "Coffee machine",
      "Washing machine",
      "Breakfast",
    ],
    host: {
      id: 25,
      name: "Angelina",
      isPro: true,
      avatarUrl: "img/avatar-angelina.jpg",
    },
    description:
      "I am happy to welcome you to my apartment in the city center! Three words: location, cosy and chic!",
    location: { latitude: 48.85761, longitude: 2.358499, zoom: 16 },
    id: 2,
  },
  {
    city: {
      name: "Cologne",
      location: { latitude: 50.938361, longitude: 6.959974, zoom: 13 },
    },
    previewImage:
      "https://htmlacademy-react-3.appspot.com/six-cities/static/hotel/3.jpg",
    images: [
      "https://htmlacademy-react-3.appspot.com/six-cities/static/hotel/6.jpg",
      "https://htmlacademy-react-3.appspot.com/six-cities/static/hotel/2.jpg",
      "https://htmlacademy-react-3.appspot.com/six-cities/static/hotel/7.jpg",
      "https://htmlacademy-react-3.appspot.com/six-cities/static/hotel/9.jpg",
      "https://htmlacademy-react-3.appspot.com/six-cities/static/hotel/11.jpg",
      "https://htmlacademy-react-3.appspot.com/six-cities/static/hotel/14.jpg",
      "https://htmlacademy-react-3.appspot.com/six-cities/static/hotel/12.jpg",
      "https://htmlacademy-react-3.appspot.com/six-cities/static/hotel/3.jpg",
      "https://htmlacademy-react-3.appspot.com/six-cities/static/hotel/1.jpg",
      "https://htmlacademy-react-3.appspot.com/six-cities/static/hotel/13.jpg",
      "https://htmlacademy-react-3.appspot.com/six-cities/static/hotel/8.jpg",
      "https://htmlacademy-react-3.appspot.com/six-cities/static/hotel/16.jpg",
      "https://htmlacademy-react-3.appspot.com/six-cities/static/hotel/10.jpg",
      "https://htmlacademy-react-3.appspot.com/six-cities/static/hotel/4.jpg",
    ],
    title: "House in countryside",
    isFavorite: false,
    isPremium: false,
    rating: 3.6,
    type: "house",
    bedrooms: 5,
    maxAdults: 5,
    price: 461,
    goods: [
      "Air conditioning",
      "Towels",
      "Laptop friendly workspace",
      "Washer",
      "Fridge",
      "Baby seat",
      "Breakfast",
    ],
    host: {
      id: 25,
      name: "Angelina",
      isPro: true,
      avatarUrl: "img/avatar-angelina.jpg",
    },
    description:
      "Peaceful studio in the most wanted area in town. Quiet house Near of everything. Completely renovated. Lovely neighbourhood, lot of trendy shops, restaurants and bars in a walking distance.",
    location: { latitude: 50.932361, longitude: 6.960974, zoom: 16 },
    id: 3,
  },
  {
    city: {
      name: "Paris",
      location: { latitude: 48.85661, longitude: 2.351499, zoom: 13 },
    },
    previewImage:
      "https://htmlacademy-react-3.appspot.com/six-cities/static/hotel/12.jpg",
    images: [
      "https://htmlacademy-react-3.appspot.com/six-cities/static/hotel/12.jpg",
      "https://htmlacademy-react-3.appspot.com/six-cities/static/hotel/5.jpg",
      "https://htmlacademy-react-3.appspot.com/six-cities/static/hotel/7.jpg",
      "https://htmlacademy-react-3.appspot.com/six-cities/static/hotel/6.jpg",
      "https://htmlacademy-react-3.appspot.com/six-cities/static/hotel/14.jpg",
      "https://htmlacademy-react-3.appspot.com/six-cities/static/hotel/4.jpg",
      "https://htmlacademy-react-3.appspot.com/six-cities/static/hotel/13.jpg",
      "https://htmlacademy-react-3.appspot.com/six-cities/static/hotel/1.jpg",
      "https://htmlacademy-react-3.appspot.com/six-cities/static/hotel/2.jpg",
      "https://htmlacademy-react-3.appspot.com/six-cities/static/hotel/10.jpg",
      "https://htmlacademy-react-3.appspot.com/six-cities/static/hotel/3.jpg",
      "https://htmlacademy-react-3.appspot.com/six-cities/static/hotel/15.jpg",
      "https://htmlacademy-react-3.appspot.com/six-cities/static/hotel/19.jpg",
      "https://htmlacademy-react-3.appspot.com/six-cities/static/hotel/11.jpg",
    ],
    title: "Perfectly located Castro",
    isFavorite: false,
    isPremium: false,
    rating: 4.1,
    type: "apartment",
    bedrooms: 5,
    maxAdults: 9,
    price: 243,
    goods: [
      "Coffee machine",
      "Air conditioning",
      "Breakfast",
      "Baby seat",
      "Dishwasher",
      "Laptop friendly workspace",
      "Fridge",
      "Washer",
      "Towels",
    ],
    host: {
      id: 25,
      name: "Angelina",
      isPro: true,
      avatarUrl: "img/avatar-angelina.jpg",
    },
    description:
      "This is a place for dreamers to reset, reflect, and create. Designed with a 'slow' pace in mind, our hope is that you enjoy every part of your stay; from making local coffee by drip in the morning, choosing the perfect record to put on as the sun sets.",
    location: { latitude: 48.861610000000006, longitude: 2.340499, zoom: 16 },
    id: 4,
  },
  {
    city: {
      name: "Brussels",
      location: { latitude: 50.846557, longitude: 4.351697, zoom: 13 },
    },
    previewImage:
      "https://htmlacademy-react-3.appspot.com/six-cities/static/hotel/18.jpg",
    images: [
      "https://htmlacademy-react-3.appspot.com/six-cities/static/hotel/5.jpg",
      "https://htmlacademy-react-3.appspot.com/six-cities/static/hotel/2.jpg",
      "https://htmlacademy-react-3.appspot.com/six-cities/static/hotel/13.jpg",
      "https://htmlacademy-react-3.appspot.com/six-cities/static/hotel/19.jpg",
      "https://htmlacademy-react-3.appspot.com/six-cities/static/hotel/9.jpg",
      "https://htmlacademy-react-3.appspot.com/six-cities/static/hotel/20.jpg",
      "https://htmlacademy-react-3.appspot.com/six-cities/static/hotel/11.jpg",
      "https://htmlacademy-react-3.appspot.com/six-cities/static/hotel/1.jpg",
      "https://htmlacademy-react-3.appspot.com/six-cities/static/hotel/18.jpg",
      "https://htmlacademy-react-3.appspot.com/six-cities/static/hotel/12.jpg",
      "https://htmlacademy-react-3.appspot.com/six-cities/static/hotel/8.jpg",
      "https://htmlacademy-react-3.appspot.com/six-cities/static/hotel/10.jpg",
      "https://htmlacademy-react-3.appspot.com/six-cities/static/hotel/14.jpg",
      "https://htmlacademy-react-3.appspot.com/six-cities/static/hotel/16.jpg",
    ],
    title: "House in countryside",
    isFavorite: false,
    isPremium: false,
    rating: 3,
    type: "house",
    bedrooms: 2,
    maxAdults: 6,
    price: 365,
    goods: ["Laptop friendly workspace"],
    host: {
      id: 25,
      name: "Angelina",
      isPro: true,
      avatarUrl: "img/avatar-angelina.jpg",
    },
    description:
      "This is a place for dreamers to reset, reflect, and create. Designed with a 'slow' pace in mind, our hope is that you enjoy every part of your stay; from making local coffee by drip in the morning, choosing the perfect record to put on as the sun sets.",
    location: { latitude: 50.842557, longitude: 4.3536969999999995, zoom: 16 },
    id: 5,
  },
];

 */
