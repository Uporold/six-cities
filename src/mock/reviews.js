export const reviews = [
  {
    id: 1,
    user: {
      id: 14,
      isPro: true,
      name: "Corey",
      avatar:
        "https://htmlacademy-react-3.appspot.com/six-cities/static/avatar/5.jpg",
    },
    rating: 4,
    comment:
      "Home is amazing. It's like staying in a museum. The rooms, furnishings and artworks are incredible. The views of My Vesuvius",
    date: "2020-07-29T07:00:10.725Z",
  },
  {
    id: 2,
    user: {
      id: 12,
      isPro: true,
      name: "Isaac",
      avatar:
        "https://htmlacademy-react-3.appspot.com/six-cities/static/avatar/3.jpg",
    },
    rating: 4,
    comment:
      "The room was spacious and clean. The pool looked nothing like the photos and desparately needs a clean. The sauna and spa were closed for lunar new year holiday.",
    date: "2020-07-29T07:00:10.725Z",
  },
];
