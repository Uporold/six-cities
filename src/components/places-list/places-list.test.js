import React from "react";
import renderer from "react-test-renderer";
import PlacesList from "./places-list";
import { hotels } from "../../mock/offers";

it(`Should Places List render correctly`, () => {
  const tree = renderer
    .create(<PlacesList hotels={hotels} onPlaceCardClick={() => {}} />, {
      createNodeMock: () => {
        return document.createElement(`div`);
      },
    })
    .toJSON();

  expect(tree).toMatchSnapshot();
});
