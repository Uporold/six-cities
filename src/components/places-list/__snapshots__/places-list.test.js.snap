// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should Places List render correctly 1`] = `
<div
  className="places__list near-places__list"
>
  <article
    className="cities__place-card place-card"
    onMouseEnter={[Function]}
  >
    
    <div
      className="cities__image-wrapper place-card__image-wrapper"
    >
      <a
        href="#"
      >
        <img
          alt="Nice, cozy, warm big bed apartment"
          className="place-card__image"
          height="200"
          onClick={[Function]}
          src="https://htmlacademy-react-3.appspot.com/six-cities/static/hotel/15.jpg"
          width="260"
        />
      </a>
    </div>
    <div
      className="place-card__info"
    >
      <div
        className="place-card__price-wrapper"
      >
        <div
          className="place-card__price"
        >
          <b
            className="place-card__price-value"
          >
            €
            488
          </b>
          <span
            className="place-card__price-text"
          >
            / night
          </span>
        </div>
        <button
          className="place-card__bookmark-button button"
          type="button"
        >
          <svg
            className="place-card__bookmark-icon"
            height="19"
            width="18"
          >
            <use
              xlinkHref="#icon-bookmark"
            />
          </svg>
          <span
            className="visually-hidden"
          >
            To bookmarks
          </span>
        </button>
      </div>
      <div
        className="place-card__rating rating"
      >
        <div
          className="place-card__stars rating__stars"
        >
          <span
            style={
              Object {
                "width": "98%",
              }
            }
          />
          <span
            className="visually-hidden"
          >
            Rating
          </span>
        </div>
      </div>
      <h2
        className="place-card__name"
      >
        <a
          href="#"
          onClick={[Function]}
        >
          Nice, cozy, warm big bed apartment
        </a>
      </h2>
      <p
        className="place-card__type"
      >
        Apartment
      </p>
    </div>
  </article>
  <article
    className="cities__place-card place-card"
    onMouseEnter={[Function]}
  >
    
    <div
      className="cities__image-wrapper place-card__image-wrapper"
    >
      <a
        href="#"
      >
        <img
          alt="Penthouse, 4-5 rooms + 5 balconies"
          className="place-card__image"
          height="200"
          onClick={[Function]}
          src="https://htmlacademy-react-3.appspot.com/six-cities/static/hotel/12.jpg"
          width="260"
        />
      </a>
    </div>
    <div
      className="place-card__info"
    >
      <div
        className="place-card__price-wrapper"
      >
        <div
          className="place-card__price"
        >
          <b
            className="place-card__price-value"
          >
            €
            327
          </b>
          <span
            className="place-card__price-text"
          >
            / night
          </span>
        </div>
        <button
          className="place-card__bookmark-button button"
          type="button"
        >
          <svg
            className="place-card__bookmark-icon"
            height="19"
            width="18"
          >
            <use
              xlinkHref="#icon-bookmark"
            />
          </svg>
          <span
            className="visually-hidden"
          >
            To bookmarks
          </span>
        </button>
      </div>
      <div
        className="place-card__rating rating"
      >
        <div
          className="place-card__stars rating__stars"
        >
          <span
            style={
              Object {
                "width": "58%",
              }
            }
          />
          <span
            className="visually-hidden"
          >
            Rating
          </span>
        </div>
      </div>
      <h2
        className="place-card__name"
      >
        <a
          href="#"
          onClick={[Function]}
        >
          Penthouse, 4-5 rooms + 5 balconies
        </a>
      </h2>
      <p
        className="place-card__type"
      >
        House
      </p>
    </div>
  </article>
  <article
    className="cities__place-card place-card"
    onMouseEnter={[Function]}
  >
    
    <div
      className="cities__image-wrapper place-card__image-wrapper"
    >
      <a
        href="#"
      >
        <img
          alt="House in countryside"
          className="place-card__image"
          height="200"
          onClick={[Function]}
          src="https://htmlacademy-react-3.appspot.com/six-cities/static/hotel/3.jpg"
          width="260"
        />
      </a>
    </div>
    <div
      className="place-card__info"
    >
      <div
        className="place-card__price-wrapper"
      >
        <div
          className="place-card__price"
        >
          <b
            className="place-card__price-value"
          >
            €
            461
          </b>
          <span
            className="place-card__price-text"
          >
            / night
          </span>
        </div>
        <button
          className="place-card__bookmark-button button"
          type="button"
        >
          <svg
            className="place-card__bookmark-icon"
            height="19"
            width="18"
          >
            <use
              xlinkHref="#icon-bookmark"
            />
          </svg>
          <span
            className="visually-hidden"
          >
            To bookmarks
          </span>
        </button>
      </div>
      <div
        className="place-card__rating rating"
      >
        <div
          className="place-card__stars rating__stars"
        >
          <span
            style={
              Object {
                "width": "72%",
              }
            }
          />
          <span
            className="visually-hidden"
          >
            Rating
          </span>
        </div>
      </div>
      <h2
        className="place-card__name"
      >
        <a
          href="#"
          onClick={[Function]}
        >
          House in countryside
        </a>
      </h2>
      <p
        className="place-card__type"
      >
        House
      </p>
    </div>
  </article>
  <article
    className="cities__place-card place-card"
    onMouseEnter={[Function]}
  >
    
    <div
      className="cities__image-wrapper place-card__image-wrapper"
    >
      <a
        href="#"
      >
        <img
          alt="Perfectly located Castro"
          className="place-card__image"
          height="200"
          onClick={[Function]}
          src="https://htmlacademy-react-3.appspot.com/six-cities/static/hotel/12.jpg"
          width="260"
        />
      </a>
    </div>
    <div
      className="place-card__info"
    >
      <div
        className="place-card__price-wrapper"
      >
        <div
          className="place-card__price"
        >
          <b
            className="place-card__price-value"
          >
            €
            243
          </b>
          <span
            className="place-card__price-text"
          >
            / night
          </span>
        </div>
        <button
          className="place-card__bookmark-button button"
          type="button"
        >
          <svg
            className="place-card__bookmark-icon"
            height="19"
            width="18"
          >
            <use
              xlinkHref="#icon-bookmark"
            />
          </svg>
          <span
            className="visually-hidden"
          >
            To bookmarks
          </span>
        </button>
      </div>
      <div
        className="place-card__rating rating"
      >
        <div
          className="place-card__stars rating__stars"
        >
          <span
            style={
              Object {
                "width": "82%",
              }
            }
          />
          <span
            className="visually-hidden"
          >
            Rating
          </span>
        </div>
      </div>
      <h2
        className="place-card__name"
      >
        <a
          href="#"
          onClick={[Function]}
        >
          Perfectly located Castro
        </a>
      </h2>
      <p
        className="place-card__type"
      >
        Apartment
      </p>
    </div>
  </article>
  <article
    className="cities__place-card place-card"
    onMouseEnter={[Function]}
  >
    
    <div
      className="cities__image-wrapper place-card__image-wrapper"
    >
      <a
        href="#"
      >
        <img
          alt="House in countryside"
          className="place-card__image"
          height="200"
          onClick={[Function]}
          src="https://htmlacademy-react-3.appspot.com/six-cities/static/hotel/18.jpg"
          width="260"
        />
      </a>
    </div>
    <div
      className="place-card__info"
    >
      <div
        className="place-card__price-wrapper"
      >
        <div
          className="place-card__price"
        >
          <b
            className="place-card__price-value"
          >
            €
            365
          </b>
          <span
            className="place-card__price-text"
          >
            / night
          </span>
        </div>
        <button
          className="place-card__bookmark-button button"
          type="button"
        >
          <svg
            className="place-card__bookmark-icon"
            height="19"
            width="18"
          >
            <use
              xlinkHref="#icon-bookmark"
            />
          </svg>
          <span
            className="visually-hidden"
          >
            To bookmarks
          </span>
        </button>
      </div>
      <div
        className="place-card__rating rating"
      >
        <div
          className="place-card__stars rating__stars"
        >
          <span
            style={
              Object {
                "width": "60%",
              }
            }
          />
          <span
            className="visually-hidden"
          >
            Rating
          </span>
        </div>
      </div>
      <h2
        className="place-card__name"
      >
        <a
          href="#"
          onClick={[Function]}
        >
          House in countryside
        </a>
      </h2>
      <p
        className="place-card__type"
      >
        House
      </p>
    </div>
  </article>
</div>
`;
